src/utils/wallet.js

```diff
@@ -446,16 +448,20 @@ const wallet = {
   async cleanWallet(wallet) {
     await wallet.storage.cleanStorage(true, true);
```

Comment:
The `cleanData` action was raising an issue, it would stop the redux wallet if it existed and clean the redux data.
By doing so, it would trigger another action and this would be an error since we cannot emit actions from reducers.

The fix for this was just making the wallet.stop call on the method that emitted the `cleanData` action which is the `cleanWalletRedux`.